frontend/src/bundles/auth/pages/auth.tsx

```diff
@@ -41,11 +63,26 @@ const Auth: React.FC = () => {
         return null;
     };
 
+    const classes = {
+        base: 'flex flex-col flex-1 bg-lm-black-200 mx-[1rem] my-[1.125rem] rounded-[2.75rem] lg:flex-none lg:w-[44rem] lg:px-[11.25rem]',
+        loader: 'justify-center items-center',
```

Comment:
You can use <Loader isOverflow /> to display it centered inside of the block. The styles of the overflow Loader use absolute position, so your parent needs to be relative. It is absolute inside parent and is already centered. You can try this

Reply:
Thanks for the advice! I will change it